### ISSUE TITLE
Equipment/Excavators quest does not complete

### ISSUE THREAD

**Issue by arisgoun** [EnigmaticaModpacks/Enigmatica9#255]
### Modpack Version

1.5

### Describe your issue.

I have the required items as shown below but the quest does not complete.
![2023-03-05_11 35 21](https://user-images.githubusercontent.com/70166657/222955404-fb22b350-4cc8-4be9-96b0-2795f48105b4.png)

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by MuteTiefling** [comment]
Thanks for the report. 

Would  you mind adding a screenshot of the quest and dependencies?

**Comment by arisgoun** [comment]
As requested:
![2023-03-05_17 41 14](https://user-images.githubusercontent.com/70166657/222973755-d9127270-5808-4901-9512-ad7abc9ad47a.png)

**Comment by MuteTiefling** [comment]
Alright. I'll just take the book off of there. Have you done the chainsaw one yet? Have the same issue?

**Comment by theboo** [comment]
First question: recover the aura in your area using some kind of aura gen.. natures aura book as well as quests should cover how to do this.. I'm assuming this happened after summoning the masticator, that quest has warnings about aura use and tells you that you should probably have some sort of aura gen first.

Second question: that quest should accept any spell parchment? If not, that sounds like some kind of bug.

Also random questions and support stuff will be better answered in our discord server! Almost always someone there to help.

https://discord.gg/enigmatica

**Comment by arisgoun** [comment]
First question: I did not summon anybody. I cannot find a way to regenerate aura, tried with creational catalyst + canopy diminisher + oak saplings to grow. No improvement ...

Second question: Yes, it does. I suppose the subtitle of the quest (Jousting Snails) is some kind of esoteric joke. Because I have been searching how to create a spell called "Jousting Snails" ...

**Comment by MuteTiefling** [comment]
Yes, jousting snails is a joke. The quest text explains what to do.

But as boo said, at this point you'd be better off coming into discord.

**Comment by arisgoun** [comment]
Regarding the breathless effect, I joined Discord (aris#7035) and read 200+ messages.
No fix.

**Comment by MuteTiefling** [comment]
Feel free to ask in Discord. These are gameplay questions and we reserve Github for bug reports.
